source/develop/release-management/features/virt/engine-dedicated-cpus.md

```diff
@@ -0,0 +1,139 @@
+---
+Title: Engine Dedicated CPUs
+Category: feature
+Authors: sabusale
+---
+
+# Engine Dedicated CPUs
+
+## Summary
+
+Once choosing dedicated CPU policy, The guest vCPUs will be strictly pinned to a set of host pCPUs
+(similarly to static CPU pinning). The set of pCPUs will be chosen to match the required guest CPU topology.
+
+Engine sets the pinning automatically when starting the VM as opposed to the manual pinning that is
+done by the user and is static.
+
+## Owner
+
+-   Name: Saif Abu Saleh (sabusale)
+-   Email: sabusale@redhat.com
+
+## Responsibilities
+
+VDSM:
+
+- Makes sure that dedicated VMs vCPUs stay exclusively pinned to the given pCPU
+
+Engine:
+- Holds cached available cpus based on host capabilities and pinned CPUs
+- Schedule VM to run on best satisfying host based on the above cached data structure
+
+
+
+## Backend
+
+1. Add dedicated to CPU Pinning Policy enum
+2. Save host capabilities CPU topology in VM Dynamic
```

Comment:
I don't get this one, you want to save host data in VM table?

Reply:
Fixed